packages/frontend/src/components/Election/ElectionStateWarning.tsx

```diff
@@ -3,18 +3,21 @@ import useElection from "../ElectionContextProvider";
 import type { ElectionState } from "@equal-vote/star-vote-shared/domain_model/Election"
 
 export default function ElectionStateWarning 
-        ({state, title, description}: {state: ElectionState, title: string, description: string}) {
+        ({state, title, description, hideIcon=false, children}: {state?: ElectionState, title: string, description: string, hideIcon?: boolean, children?: any}) {
     
     const { t, election } = useElection();
     
     if(election.state !== state) return <></>
 
-    return <Paper sx={{display: 'flex', flexDirection: 'row', maxWidth: 600, gap: 2, padding: 2, m: 'auto', mb:4}}>
-        <Typography component="h3">⚠️</Typography>
-        <Box>
-            <Typography component="p"><b>{t(title)}</b></Typography>
-            <hr/>
-            <Typography component="p">{t(description)}</Typography>
+    return <Paper sx={{display: 'flex', flexDirection: 'column', maxWidth: 600, gap: 2, padding: 2, m: 'auto', mb:4}}>
+        <Box display='flex' flexDirection='row' gap={2} sx={{p: 2, m: 'auto'}}>
+            {!hideIcon && <Typography component="h3">⚠️</Typography>}
```

Comment:
It's outside the scope of this feature, but it seems like we could use an MUI icon for this

Reply:
Sure, that's quick. Here's a comparison. I also did a search and we don't use that emoji anywhere beside the email template (which I think doesn't support mui icons)

<img width="661" height="191" alt="Image" src="https://github.com/user-attachments/assets/a3537cb2-8af8-470d-afdf-6d207a556f27" />

<img width="575" height="152" alt="Image" src="https://github.com/user-attachments/assets/22f01e9c-aaae-4655-8391-baa3f0dc2094" />